lib/src/doc/table.rs

```diff
@@ -261,67 +261,40 @@ impl<'a> Document<'a> {
 		};
 		//
 		for field in exp.other() {
-			// Process it if it is a normal field
-			if let Field::Alone(v) = field {
-				match v {
+			// Process the field
+			if let Field::Single {
+				expr,
+				alias,
+			} = field
+			{
+				let idiom = alias.clone().unwrap_or_else(|| expr.to_idiom());
```

Comment:
@finnbear up above you changed this line to...
```rs
let idiom = alias
    .as_ref()
    .map(Cow::Borrowed)
    .unwrap_or_else(|| Cow::Owned(expr.to_idiom()));
```
Should this be the same as above? Currently the `clone()` is cloning it, but the `to_idiom()` takes a reference, and is likely cloning or outputting to string. So effectively we have a duplicate clone?

Reply:
I think it's fine as-is, since only one costly clone is performed at runtime. If `alias` is `Some`, the first `clone` will allocate a clone. If `alias` is `None`, the first clone is basically a no-op and the second `clone` (`to_idiom`) will allocate. Using a `Cow` here doesn't help since the downstream code needs to take ownership.